ts/Extensions/Breadcrumbs.ts

```diff
@@ -1097,6 +1028,65 @@ if (!H.Breadcrumbs) {
     });
 }
 
+/* *
+ *
+ *  Prototype Properties
+ *
+ * */
+
+interface Breadcrumbs {
+    breadcrumbsGroup: SVGElement;
+    breadcrumbsList: Array<Array<any>>;
+    chart: Chart;
+    isDirty: boolean;
+    isTreemap?: boolean;
+    level: number;
+    options: BreadcrumbsOptions;
+}
```

Comment:
The properties of interface `Breadcrumbs` should move to the class and the interface itself deleted.